sample-project/app/models/user.rb

```diff
@@ -1,2 +1,3 @@
 class User < ApplicationRecord
+  foo = 1
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vividmuimui/danger-sample/blob/a7af05ac2a749028a5052fb1251691ee29746076/sample-project/app/models/user.rb#L2"></span>Lint/UselessAssignment: Useless assignment to variable - <code>foo</code>.</td>
    </tr>
  </tbody>
</table>